packages/shared/src/libs/helpers/get-formatted-date/get-formatted-date.helper.ts

```diff
@@ -2,10 +2,8 @@ import { format } from "date-fns";
 
 type DateFormat = "d MMM yyyy, EEEE";
 
-const getFormattedDate = (dateFormat: DateFormat): string => {
-	const today = new Date();
-
-	return format(today, dateFormat);
+const getFormattedDate = (dateFormat: DateFormat, date: string): string => {
```

Comment:
```suggestion
const getFormattedDate = (date: Date, dateFormat: DateFormat): string => {
```
That makes more sense